listen.js

```diff
@@ -0,0 +1,9 @@
+const app = require("./app");
```

Comment:
We don't really need a listen.js right now as Supertest is handling some of the required functionality whilst we're testing - however you'll need this when you host your application (more on that later this week!)